script/brave_license_helper.py

```diff
@@ -132,6 +132,19 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
             "URL": "https://github.com/brave/Sparkle",
             "License": "MIT",
         },
+        os.path.join('brave', 'third_party', 'cryptography'): {
+            "Name": "cryptography",
+            "URL": "https://cryptography.io",
+            "License": "Apache-2.0",
+            # Because of the way we install this third-party library, there is
```

Comment:
nit: this comment is not really necessary because that's the case for lots of other files in brave-core

Reply:
Resolved in de8d9b061b34f572537da367f574c3a64e52c89c.